run_cpu_exp3.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python3
+
+from models.tensorflow.Lenet5 import Lenet5
+from ratelimit_stream import construct_randomsleep_rows
```

Comment:
Please do not use this package to generate data. This package should be deprecated. 

Please use the `stream_utils.ImageGenerator` package to generate your images / labels. There is a main code there that tells you how to use it.